Simplified/en.lproj/Localizable.strings

```diff
@@ -270,3 +270,9 @@
 
 // Announcement
 "Announcement" = "Announcement";
+
+// Library Finder
+"Find a Library" = "Find a Library";
+"LibraryListUpdateFailed" = "Failed to update library list";
+"Location, library name or zip code" = "Location, library name or zip code";
```

Comment:
Should we use oxford comma here e.g. `Location, library name, or zip code`? @ettore Do you guys use oxford comma in states?

Reply:
oh I don't know, I'm not from here 😬 but I think so? This is a good question for @killerpuppytails :Laura

Reply:
Thanks for tagging me @ettore! @singhraman4282 great call pointing this out. In the US some organizations do use Oxford comma and some don't; it's left up to individual institutions. NYPL works hard to avoid constructing sentences in a way where this comes up! In this context I believe the text should be changed because Location is comprehensive of ZIP code. (Also, ZIP is all caps since it's an acronym.) Slack me where this should show up and I'll verify a different chunk of text.